repositories/riotrepository.go

```diff
@@ -0,0 +1,97 @@
+package repositories
```

Comment:
I'm not really sure what "repository" means in this context; would it make more sense to call this `summoner.go` or something like that, and have the folder name be something like `apis` or `rgapis`?

Reply:
Sure! I renamed the folder as `rgapis` and this file/struct as `RGAPIWrapper`